### PR TITLE
Fix: CORS logging issue #1576

### LINKED ISSUE #1576: Cors failure logs too much
```
Jul 11 17:31:16 Error: http://localhost:3002 not allowed at corsOptions.origin (/usr/lib/node_modules/signalk-server/lib/cors.js:42:20) at /usr/lib/node_modules/signalk-server/node_modules/cors/lib/index.js:219:13 at optionsCallback (/usr/lib/node_modules/signalk-server/node_modules/cors/lib/index.js:199:9) at corsMiddleware (/usr/lib/node_modules/signalk-server/node_modules/cors/lib/index.js:204:7) at Layer.handle [as handle_request] (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/layer.js:95:5) at trim_prefix (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:328:13) at /usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:286:9 at Function.process_params (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:346:12) at next (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:280:10) at Layer.handle [as handle_request] (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/layer.js:91:12) at trim_prefix (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:328:13) at /usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:286:9 at Function.process_params (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:346:12) at next (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:280:10) at logger (/usr/lib/node_modules/signalk-server/node_modules/morgan/index.js:144:5) at Layer.handle [as handle_request] (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/layer.js:95:5)
```

### PR DESCRIPTION
**Pull request by JacoKoster**
Fixes #1576

### CODE REVIEW

File: src/cors.ts [hunk 1]
```diff
@@ -18,23 +15,11 @@ export function setupCors(
         .map((s: string) => s.trim().replace(/\/*$/, ''))
```
Comment (tkurki) [inline src/cors.ts:15]: This processing is now dead code. Will trailing slashes now work or not?

**Review by tkurki — COMMENTED** [review]
With this change there's no logging of failed requests, right? I have found that useful, as CORS failures are so opaque otherwise.

Setting up CORS is kinda painful. I was thinking the server could have a feature where the admin UI shows CORS failures with a button to add the origin.

**Comment by tkurki** [comment]
Do you plan to continue with this?

**Comment by JacoKoster** [comment]
Yes, i will probably pick it up again when i'm back from sailing, unless you want to pick it up, then we can close it ;-)

**Comment by tkurki** [comment]
😄 the amount and structure of the code is almost exactly the same as the original.

**Comment by tkurki** [comment]
I reworded the PR title a little as release notes are created from PRs, not issues, so they should be self documenting when possible.

Fix or feature - or is it just a change or improvement? In the end it is in the eye of the reader 🤷 .

**Pull request merged**